spec/v2/index.bs

```diff
@@ -2224,6 +2226,16 @@ path: LICENSE.md
 # Appendix B: Changelog # {#changelog}
 
 
+## Version 2.0.1-20230925 (Sep 25, 2023) ## {#changelog-2.0.1-20230925}
+
+Summary of changes:
+
+1. a definition fix on the definedness for properties <{CarbonFootprint/primaryDataShare}> and <{CarbonFootprint/dqi}>: 
+    previously, they were defined in a mutual-exclusive fashion (either one must be defined but *NOT* both) while the Pathfinder Framework Version 2.0 defines them as follows (Section 4.2.1, Page 39):
+    ```Initially, companies shall calculate and report, as part of PCF data exchange, on at least one of the following metrics: [...]```  
+
```

Comment:
Explanation makes sense! I will share two english corrections, not to be nit-picky but since I'm here may as well. "definedness" isn't a commonly understood or used word, would just say "a definition fix for the properties"...

Second, would change "mutual-exclusive" to "mutually-exclusive", as that's the expression in English, no idea why we say "mutually" rather than "mutual", but I guess that's the challenge of standardizing language

Reply:
reformulated this section with 1bf3d2fad22fb55de0b0e057a87400460a69b78a. Please review @bhadley

Reply:
reformulated this section with 1bf3d2fad22fb55de0b0e057a87400460a69b78a. Please review @bhadley